src/mailer/alerts.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+src_dir = os.path.abspath(os.path.join(current_dir, ".."))
+p4hApp_dir = os.path.abspath(os.path.join(src_dir, ".."))
+sys.path.append(p4hApp_dir)
```

Comment:
```suggestion
p4hApp_dir = os.path.abspath(os.path.join(current_dir, "../../"))
sys.path.append(p4hApp_dir)
```
Would something like this be possible?